openedx/course-discovery
# feat: use config model for csv file and update import degree metadata

[PROD-2814](https://2u-internal.atlassian.net/browse/PROD-2814)
Using a config model instead of using the csv file path in `import_degree_data` command. A file can be uploaded on the config model through django admin which will be used in the current version of the `import_degree_data` command. We want to be able to run this command on Jenkins. We opted to keep the config models separate so that we will be able to run all of the ingestions separately.

We will be adding this command in our stage/prod jenkins in the following PR: https://github.com/edx/edx-internal/pull/6828

### Testing Instructions
1. Run migrations
2. Add an entry against `http://localhost:18381/admin/course_metadata/degreedataloaderconfiguration/` with a sample degree data file
3. Run `./manage.py import_degree_data` in Discovery Shell
4. Expect it to run the same way it does with `./manage.py import_degree_data --csv_path=degree.csv`
